scripts/CEE/delete-os-pod/README.md

```diff
@@ -0,0 +1,21 @@
+# Delete Openshift Pod Script
```

Comment:
Can we simplify this to just be `delete-pod` instead of adding the `delete-os-pod`? From a UX perspective, it will be easier to remember the closer the syntax name is to the actual OC command.